src/Finder/LscpuFinder.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Fidry CPUCounter Config package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\CpuCoreCounter\Finder;
+
+use function sprintf;
+
+/**
+ * The number of cores.
+ *
+ * @see https://stackoverflow.com/a/23378780/5846754
+ */
+final class LscpuFinder extends ProcOpenBasedFinder
+{
+    /**
+     * @var bool
+     */
+    private $physical;
+
+    public function __construct(bool $physical = false)
+    {
+        $this->physical = $physical;
+    }
+
+    public function getCommand(): string
+    {
+        return sprintf(
+            "lscpu -p | egrep -v '^#' | %s wc -l",
+            $this->physical ? 'sort -u -t, -k 2,4 |' : ''
+        );
+    }
+
+    public function toString(): string
+    {
+        return 'LscpuFinder';
```

Comment:
I would reflect the `physical` property there, you can check `NprocFinder` as a reference

Reply:
This has changed with the splitting to two finders.